skbio/stats/distance/tests/test_base.py

```diff
@@ -542,6 +542,9 @@ def test_default_usage(self):
 
         self.assertTrue(found_diff)
 
+        obs3 = randdm(100)
+        self.assertTrue(obs3 == obs3.T)
```

Comment:
Maybe `self.assertEqual(obs3, obs3.T)` would be more clear.

Also, it should live in its own method with a descriptive name (say, `def test_large_matrix_for_symmetry`) so that when it fails we know exactly what failed, as opposed to big test methods where a failure doesn't really tell you what exactly failed.